dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/FindableObjectRepository.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.repository;
+
+import java.io.Serializable;
+import java.sql.SQLException;
+
+import org.dspace.core.Context;
+import org.dspace.core.ReloadableEntity;
+
+/**
+ * This interface must be implemented by all the rest repository that need to
+ * provide access to the DSpace API model objects corresponding to the REST
+ * resources that it manages
+ * 
+ * @author Andrea Bollini (andrea.bollini at 4science.it)
+ *
+ * @param <F> the ReloadableEntity type
```

Comment:
This <F>, should be <T>

Reply:
This is now the ReloadableEntityObjectRepository, I guess you noted that F was used instead than T, now fixed